Documentation/io.mender.Authentication1.xml

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+
+<node>
+  <!--
+    io.mender.Authentication1:
+    @short_description: Mender Authentication API v1
+
+    This interface lets applications authenticate with the Mender server. The
+    Mender client will handle the authentication and provide to the user with a
+    JSON Web Token (JWT) and the server URL, which which the user can do its own
+    API calls to the Mender server.
+  -->
+  <interface name="io.mender.Authentication1">
+
+    <!--
+      GetJwtToken:
+      @token: Valid JWT token
+      @server_url: Server URL
+
+      Gets current JWT token and server URL. If no JWT token is available, error
+      is returned.
```

Comment:
Gets current JWT token and server URL. If no JWT token is available, an error
is returned.

Reply:
Gets current JWT token and server URL. If no JWT token is available, **an empty token**
is returned.